tests/e2e/unified_test.go

```diff
@@ -711,3 +604,118 @@ func addTestTimeout(ctx context.Context, t *testing.T, timeout time.Duration) co
 
 	return ctx
 }
+
+func vcrConfig(t *testing.T, h *create.Harness) {
```

Comment:
Nit: Maybe `configureVCR`?  (Because `vcrConfig` sounds more like a function that returns the vcr configuration, rather than a function that sets the VCR configuration)

Reply:
Done